doc/semitrans.xml

```diff
@@ -349,3 +349,29 @@ gap> FixedPointsOfTransformationSemigroup(S);
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="WreathProduct">
+  <ManSection>
+    <Oper Name = "WreathProduct" Arg = "M, S"/>
+    <Returns>A transformation semigroup.</Returns>
+    <Description>
+      If <A>M</A> is a transformation monoid or a permutation group, and
+      <A>S</A> is a transformation semigroup or a permutation group, the
+      function <C>WreathProduct(<A>M</A>, <A>S</A>)</C> outputs the wreath
```

Comment:
`the function <C>WreathProduct(<A>M</A>, <A>S</A>)</C> outputs` -> `the operation <C>WreathProduct</C> returns` for consistency with the rest of the manual.